app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -36,6 +36,8 @@
   flex: 1 0 auto;
   padding: $su-4;
   width: 100%;
+  max-height: calc(100vh - 70px);
+  overflow: scroll;
```

Comment:
Hm, I'm not sure we should be putting this on `onboarding-content`. Seems like it affects every page in the modal. Maybe we can add a specific class to the profile page of the onboarding flow and use it there?